test/specs/babel-eslint.js

```diff
@@ -13,12 +13,15 @@ function lookup(obj, keypath, backwardsDepth) {
     return obj;
   }
 
-  return keypath
-    .split(".")
-    .slice(0, -1 * backwardsDepth)
-    .reduce((base, segment) => {
-      return base && base[segment], obj;
-    });
+  let pathSegments = keypath.split(".");
+
+  if (pathSegments.length > 1) {
```

Comment:
There was a bug here where if it's a top level key (i.e. `start`) it would throw and the reported error was a runtime error rather than an assertion error.